modules/@apostrophecms/doc-type/ui/apos/components/AposDocMoreMenu.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <AposContextMenu
+    class="apos-admin-bar__context-button"
+    :menu="menu"
+    menu-placement="bottom-end"
+    @item-clicked="menuHandler"
+    :button="{
+      tooltip: { content: 'More Options', placement: 'bottom' },
+      label: 'More Options',
+      icon: 'dots-vertical-icon',
+      iconOnly: true,
+      type: 'subtle',
+      modifiers: ['small', 'no-motion']
+    }"
+  />
+</template>
+
+<script>
+
+export default {
+  name: 'AposDocMoreMenu',
+  props: {
+    docId: {
+      type: String,
+      required: true
+    }
+  },
+  emits: [ 'admin-menu-click', 'close' ],
+  data() {
+    return {
+      isOpen: false,
+      menu: [
+        {
+          label: 'Share Draft',
+          action: 'share'
+        },
+        {
+          label: 'Save Draft',
+          action: 'save'
+        },
+        {
+          label: 'Duplicate Document',
+          action: 'duplicate'
+        },
+        {
+          label: 'Version History',
+          action: 'versions'
```

Comment:
This does not exist, but we are going to have to make this list extensible by pushing it from the server via browserOptions soon (not a blocker for now).